pkg/kubectl/cmd/update.go

```diff
@@ -88,27 +104,27 @@ func updateWithPatch(cmd *cobra.Command, args []string, f *Factory, patch string
 	return name
 }
 
-func updateWithFile(cmd *cobra.Command, f *Factory, filename string) string {
+func updateWithFiles(cmd *cobra.Command, f *Factory, info *resource.Info) error {
```

Comment:
I find this name confusing. Isn't this just one object, the source of which is abstracted? How about `updateObject`?